web/build.gradle.kts

```diff
@@ -150,7 +150,7 @@ tasks {
   bootRun {
     systemProperty("dev.http.port", devPort)
     systemProperty("dev.http.port.internal", devPortInternal)
-    systemProperty("spring.profiles.active", "dev, fake, caching-enabled, it-stub, wc-stub, pp-stub, ts-stub")
+    systemProperty("spring.profiles.active", "dev, fake, caching-enabled, it-stub, wc-stub, pp-stub")
```

Comment:
Perhaps we should keep the ts-stub as default?

Reply:
Commited it by mistake :)
Fixed!